src/test/regress/expected/view_propagation.out

```diff
@@ -0,0 +1,320 @@
+SELECT * FROM (SELECT pg_identify_object_as_address(classid, objid, objsubid) as obj_identifier from pg_catalog.pg_dist_object) as obj_identifiers where obj_identifier::text like '%opt_prop_view%';
+                    obj_identifier
+---------------------------------------------------------------------
+ (view,"{view_prop_schema,aliased_opt_prop_view}",{})
+ (view,"{view_prop_schema,opt_prop_view}",{})
+ (view,"{view_prop_schema,sep_opt_prop_view}",{})
+(3 rows)
+
+-- Check definitions of views are correct on workers
+\c - - - :worker_1_port
+SET search_path to view_prop_schema;
+\d+ aliased_opt_prop_view
+                View "view_prop_schema.aliased_opt_prop_view"
+ Column  |  Type   | Collation | Nullable | Default | Storage  | Description
+---------------------------------------------------------------------
+ alias_1 | integer |           |          |         | plain    |
+ alias_2 | text    |           |          |         | extended |
+View definition:
+ SELECT view_table_6.id AS alias_1,
+    view_table_6.val_1 AS alias_2
+   FROM view_table_6;
+
+\d+ opt_prop_view
+                   View "view_prop_schema.opt_prop_view"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description
+---------------------------------------------------------------------
+ id     | integer |           |          |         | plain    |
+ val_1  | text    |           |          |         | extended |
+View definition:
+ SELECT view_table_6.id,
+    view_table_6.val_1
+   FROM view_table_6;
+Options: check_option=cascaded, security_barrier=true
+
+\d+ sep_opt_prop_view
+                 View "view_prop_schema.sep_opt_prop_view"
+ Column |  Type   | Collation | Nullable | Default | Storage  | Description
+---------------------------------------------------------------------
+ id     | integer |           |          |         | plain    |
+ val_1  | text    |           |          |         | extended |
+View definition:
+ SELECT view_table_6.id,
+    view_table_6.val_1
+   FROM view_table_6;
+Options: check_option=local
+
+-- Drop views and check metadata afterwards
+\c - - - :master_port
+SET search_path to view_prop_schema;
+DROP VIEW prop_view_9 CASCADE;
+NOTICE:  drop cascades to view prop_view_10
+DROP VIEW opt_prop_view, aliased_opt_prop_view, view_prop_schema_inner.inner_view_prop, sep_opt_prop_view;
+SELECT * FROM (SELECT pg_identify_object_as_address(classid, objid, objsubid) as obj_identifier from pg_catalog.pg_dist_object) as obj_identifiers where obj_identifier::text like '%inner_view_prop%';
+ obj_identifier
+---------------------------------------------------------------------
+(0 rows)
+
+SELECT * FROM (SELECT pg_identify_object_as_address(classid, objid, objsubid) as obj_identifier from pg_catalog.pg_dist_object) as obj_identifiers where obj_identifier::text like '%opt_prop_view%';
+ obj_identifier
+---------------------------------------------------------------------
+(0 rows)
+
+DROP SCHEMA view_prop_schema_inner CASCADE;
```

Comment:
should we add some tests: 
- DROP COLUMN that a view depends
- DROP TABLE that a view depends

And, did we add the metadata consistency check to the regression tests? (e.g., are we verifying that the pg_dist_object are in sync on all nodes)

Reply:
I've added tests related to drop column and table. And yes we have metadata consistency check at the end of the schedule